src/plays/service/PlaysService.ts

```diff
@@ -27,6 +32,15 @@ export default class PlaysService {
     });
     return deferred.promise as any;
   }
+  public create(play: PlayCreateRequest): Promise<Play> {
+    const deferred = Q.defer<Play | Play[]>();
+    const requestUrl = `/${Version.V2}${Resource.PLAYS}`;
+    this.client.put(requestUrl, { data: play }, (error, response, body) => {
```

Comment:
@appu-channelape this should be POST, not PUT. Please fix it to be the correct method for creating plays

Reply:
oh good catch @ryankazokas thank you

Reply:
To catch small things like this before pushing, you can E2E test by creating a simple local JS file and then installing your local channelape-sdk using `npm link`.

I have a local "sandbox" folder with a package.json and index.js file with the following which I use to test my SDK changes:
```
const channelApeSdk = require('channelape-sdk');
const client = new channelApeSdk.ChannelApeClient({
	minimumRequestRetryRandomDelay: 1000,
	maximumRequestRetryRandomDelay: 3000,
	maximumRequestRetryTimeout: 3000,
	sessionId: SESSION_ID,
	maximumConcurrentConnections: 10
});
run();
async function run() {
	try {
		const res = await client.plays().get();
		console.log(res);
	} catch (error) {
		console.log(error);
	}
	console.log('done');
}
```

I call `node index.js` to run it.

Some instructions I pulled and modified from our web app repo regarding using `npm link`:
To use changes you have made in the your local channelape-sdk repository before they are published to NPM, follow these steps:

1. In the channelape-sdk:
    1. Run `npm run compile` (important - must be done after every change you make to the sdk repo)
    2. Run `npm link`
2. In the repo you wish to use your local channelape-sdk version:
    1. Run `npm link channelape-sdk`
    
If you need to stop referencing the local channelape-sdk version, run `npm unlink channelape-sdk`.